lib/ts/recipe/dashboard/api/implementation.ts

```diff
@@ -54,6 +59,51 @@ export default function getAPIImplementation(): APIInterface {
                 isSearchEnabled = true;
             }
 
+            const loginMethods = {
```

Comment:
this is all not supposed to be here, cause last we discussed that the dashboard will be calling the dashboard api to get the login method. So why is this still here?

Reply:
We would actually be needing both this and the dashboard API, because they both serve different purposes:

- The dashboard API will be used at the tenant listing page and the user creation dashboard to check all the login methods available based on first factors for different tenants

- Whereas we still need to know which recipes are initialized in the backend SDK, so that we can use this information to decide which login methods can be enabled in the tenant details page in frontend